README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+---
+output:
```

Comment:
README files are rendered automatically to GitHub; so you don't need to add this form of YAML metadata. These will be necessary for `.Rmd` files for future homeworks, but not for the READMEs.